mantidproject/mantid
# Unscripted tests for SANS GUI v2

**Description of work.**

**Report to:** nobody

**To test:**

* Build a dev docs target of Mantid `dev-docs-html`
* In `dev-docs/html/Testing` open `index.html`
* Check that there is a link to `SANS GUI Testing`
* Follow the link and check that the following page makes sense

*There is no associated issue.*

*This does not require release notes* because it is not a user-facing change.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
